Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -126,75 +199,43 @@ public States States
         /// </summary>
         public string VoiceCommand = "";
```

Comment:
>     public string VoiceCommand = ""; [](start = 3, length = 37)

not required to fix in this PR, but support for multiple voicecommands is useful. Some apps configure shell parity "Select" and a global voice command configured to trigger the same interactable

Reply:
That is an interesting point. Definitely not going to do in this PR to mitigate risk but @ritijain open a bug and feel free to link in the description of this PR with the other issues I have found